### DISCUSSION TITLE
Tab insertion formatting issue in Visual Studio

### DISCUSSION THREAD

**Discussion by davidnmbond** [community/community#66684] [Copilot Conversations]
### Select Topic Area

Bug

### Body

**Description**
When inserting a suggestion split across multiple lines with the user's whitespace preference set to tabs (via .editorconfig), the preview shows it perfectly as it should be.  However when actually inserting, additional whitespace tabs are added that must then be manually removed.

**Steps to reproduce**
See screenshot:
![GitHub Copilot Issue](https://github.com/community/community/assets/7120378/741d8ea4-6a4d-40f3-9d78-1398b3022472)

**Expected behavior**
The code should be inserted as per the preview.

**Comment by LockTar** [comment]
I have version VS2022 17.7.5 and problem is still there.
